.github/workflows/pr-labeler.yml

```
@@ -22,6 +22,7 @@ jobs:
               '[test]': '✅ test',
               '[utils]': '⚙️ utils'
             };
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)